src/Sylius/Bundle/WebBundle/Resources/views/Frontend/Product/macros.html.twig

```diff
@@ -32,3 +32,9 @@
 </div>
 
 {% endmacro %}
+
+{% macro price(variant, classes) %}
+{% if (variant.originalprice > variant.price) %}
+    <span class="label crossed {% for class in classes %} {{ class }} {% endfor %}">{{ variant.originalPrice|sylius_price }}</span>
+{% endif %}
+{% endmacro %}
```

Comment:
I would do this macro a bit differently. I don't think passing in classes achieves much except handling the 'pull-right' case which I think could be done differently:

```
{% macro sylius_display_price_with_original(variant) %}
    {% if (variant.isReduced) %}
        <span class='label label-success original-price'>{{ variant.originalPrice|sylius_price }}</span>
    {% endif %}
    <span class='label label-success'>{{variant.price|sylius_price }}</span>
{% endmacro %}
```

This way your macro removes any context of the position of the label - it just puts semantic mark-up around it that can be styled as required.

If you wanted to use this with a 'pull-right' in some cases then just wrap it:

```
<span class="pull-right">{{ sylius_display_price_with_original(variant) }}</span>
```

Which would generate:

```
<span class="pull-right">
    <span class="label label-primary original-price">€25</span>
    <span class="label label-primary">€15</span>
</span>
```

It does mean we assume mark-up in a macro, but it's the same mark-up everywhere so makes sense to me...